lib/config/file.ts

```diff
@@ -36,13 +36,15 @@ export const readConfigFile = (
     return tryLoad(filePathFromArgs);
   }
 
+  const packageDirectory = packageDirectorySync();
+
   const filePath = [
-    '/.spellcheckerrc.yaml',
-    '/.spellcheckerrc.yml',
-    './spellcheckerrc.json',
-    './spellcheckerrc.jsonc',
+    './.spellcheckerrc.yaml',
+    './.spellcheckerrc.yml',
+    './.spellcheckerrc.json',
+    './.spellcheckerrc.jsonc',
```

Comment:
Whoops, these were not the filenames mentioned in the README. This is a bugfix IMO.